controller.js

```diff
@@ -87,6 +87,22 @@ const updateArticleVotes = (req, res, next) => {
     })
     .catch(next);
 };
+//CORE TASK-9: DELETE /api/comments/:comment_id
+const deleteComment = (req, res) => {
+  const { comment_id } = req.params;
+  deleteCommentModel(comment_id)
+    .then(() => {
+      res.status(204).send(); // No Content
+    })
+    .catch((err) => {
+      if (err.code === "23503") {
+        return res.status(404).send({ msg: "Comment not found" });
+      } else {
+        return res.status(500).json({ msg: "Internal Server Error" });
+      }
+    });
```

Comment:
Have a read of the error handling notes on a way to refactor this to use next and middleware:
https://l2c.northcoders.com/courses/sd-notes/back-end#sectionId=error-handling,step=intro